specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -2,120 +2,150 @@
 
 # Background
 
-The Windows Push Notification Service (WNS) enables third-party developers to send toast, tile, badge, <br/>
-and raw updates from their own cloud service. This provides a mechanism to deliver new updates to your <br/>
-users in a power-efficient and dependable way. For example, the skype app service can triggering an incoming <br/>
-call toast by Pushing a Payload to the WNS service. Push is generally used for time critical <br/>
-events that require immediate user/developer action.
-
-* [The Push notification WinRT Public APIs](https://docs.microsoft.com/uwp/api/windows.networking.pushnotifications?view=winrt-19041)
-  Defines the classes that encapsulate Push notifications. (UWP and Win32)
-* [The Windows Push Notification Services (WNS) Overview](https://docs.microsoft.com/windows/uwp/design/shell/tiles-and-notifications/windows-push-notification-services--wns--overview) Defines how an app today can subscribe to Push Notifications and send a payload down to the corresponding client.
+The Windows Push Notification Service (WNS) enables third-party developers to send toast, tile,
+badge, <br/> and raw updates from their own cloud service. This provides a mechanism to deliver new
+updates to your <br/> users in a power-efficient and dependable way. For example, the skype app
+service can triggering an incoming <br/> call toast by Pushing a Payload to the WNS service. Push is
+generally used for time critical <br/> events that require immediate user/developer action.
+
+-   [The Push notification WinRT Public APIs](https://docs.microsoft.com/uwp/api/windows.networking.pushnotifications?view=winrt-19041)
+    Defines the classes that encapsulate Push notifications. (UWP and Win32)
+-   [The Windows Push Notification Services (WNS) Overview](https://docs.microsoft.com/windows/uwp/design/shell/tiles-and-notifications/windows-push-notification-services--wns--overview)
+    Defines how an app today can subscribe to Push Notifications and send a payload down to the
+    corresponding client.
 
 The below diagram is an illustration of the overall Push Flow that we have today:<br/>
 ![Legacy Flow For Windows App SDK](Legacy.png)
 
 ## The problems today
 
-**Portal Registration**: The entire Registration flow for Push is dependent on registering an app in the <br/>
-partner center portal by [associating an app with the Windows Store](https://docs.microsoft.com/azure/notification-hubs/notification-hubs-windows-store-dotnet-get-started-wns-push-notification). As a result, non-store provisioned <br/>
-apps cannot use this feature.
+**Portal Registration**: The entire Registration flow for Push is dependent on registering an app in
+the <br/> partner center portal by
+[associating an app with the Windows Store](https://docs.microsoft.com/azure/notification-hubs/notification-hubs-windows-store-dotnet-get-started-wns-push-notification).
+As a result, non-store provisioned <br/> apps cannot use this feature.
```

Comment:
Can you make a pass and see what's going on with all these `<br/>` in the document? Markdown automatically generates appropriate linebreaks. If you want a new paragraph, just use a blank like:

```
instead of <br/> this style
```

```
use this

style.
```